src/arrayMethodSort.js

```diff
@@ -4,9 +4,45 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    a.toString() > b.toString()) {
+    for (let i = 1; i < this.length; i++) {
+      for (let j = 0; j < this.length - 1; j++) {
+        if (compareFunction(this[j], this[j + 1]) > 0) {
+          const temp = this[j];
+
+          this[j] = this[j + 1];
+          this[j + 1] = temp;
+        }
+      }
+    }
+
+    return this;
   };
 }
 
+// function applyCustomSort() {
+//   [].__proto__.sort2 = function(compareFunction) {
+//     for (let i = 1; i < this.length; i++) {
+//       for (let j = 0; j < this.length - 1; j++) {
+//         if (compareFunction) {
+//           if (compareFunction(this[j], this[j + 1]) > 0) {
+//             const temp = this[j];
+
+//             this[j] = this[j + 1];
+//             this[j + 1] = temp;
+//           }
+//         } else if (this[j].toString() > this[j + 1].toString()) {
+//           const temp = this[j];
+
+//           this[j] = this[j + 1];
+//           this[j + 1] = temp;
+//         }
+//       }
+//     }
+
+//     return this;
+//   };
+// }
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)